src/arfsdao_anonymous.ts

```diff
@@ -55,6 +63,14 @@ export class ArFSDAOAnonymous extends ArFSDAOType {
 		}
 
 		const edgeOfFirstDrive = edges[0];
+
+		if (drivePrivacy) {
+			// Assert drive privacy tag if it has been passed down
+			if (edgeOfFirstDrive.node.tags.find((t) => t.name === 'Drive-Privacy')?.value !== drivePrivacy) {
```

Comment:
It would be problematic if there were no Drive-Privacy tag. Perhaps we should error on that?

Reply:
In that case it will error here, `undefined` will not equal `drivePrivacy`

Reply:
But we could give a more specific error, adding one

Reply:
23e06bc